dbqueries.py

```diff
@@ -114,11 +122,59 @@ class QueryDict(TypedDict):
             }
         }
     },
+    'udw_duplicate_course_ids': {
+        'output_file_name': 'udw_duplicate_course_ids.csv',
+        'data_source': 'UDW',
+        'query_name': 'UDW Duplicate Course IDs',
+        'type': 'standard',
+        'query': '''
+            select c.canvas_id, count(c.canvas_id)
+            from course_dim c
+            left join enrollment_term_dim t
+                on t.id=c.enrollment_term_id
+            where c.workflow_state != 'deleted'
+                and t.date_start <= current_timestamp
+                and t.date_start + interval '120 days' >= current_timestamp
+            group by c.canvas_id
+            having count(*) > 1;
+        ''',
+        'checks': {
+            'less_than_two': {
+                'color': 'RED',
+                'condition': LESS_THAN_TWO,
+                'rows_to_ignore': []
+            }
+        }
+    },
+    'udw_duplicate_assignment_ids': {
+        'output_file_name': 'udw_duplicate_assignment_ids.csv',
+        'data_source': 'UDW',
+        'query_name': 'UDW Duplicate Assignment IDs',
+        'type': 'standard',
+        'query': '''
+            select c.canvas_id, count(c.canvas_id)
```

Comment:
It *looks* like this is exactly the same query as the course query in `udw_duplicate_course_ids`. Was this intended to be checking `assignment_dim`?

Reply:
Wups! I'll fix that.

Reply:
I think I had the query written, just must not have copied it over correctly.

Reply:
Fixed. https://github.com/tl-its-umich-edu/unizin-validation/pull/34/commits/7d65a10a6a3f706b9f49bed18348466f0b25d049 Sorry, pretty bad mistake, thanks for catching!